packages/primitives/src/Button.tsx

```diff
@@ -204,8 +204,8 @@ export type ButtonProps = BaseButtonProps & ButtonVariantProps;
 const StyledButton = styled(ark.button, {}, { baseComponent: true, defaultProps: { type: "button" } });
 
 export const BaseButton = forwardRef<HTMLButtonElement, BaseButtonProps>(
-  ({ loading, loadingContent: loadingContentProp, replaceContent, onClick, children, ...props }, ref) => {
-    const ariaDisabled = loading ? { "aria-disabled": true } : {};
+  ({ loading, disabled, loadingContent: loadingContentProp, replaceContent, onClick, children, ...props }, ref) => {
+    const trulyDisabled = disabled || loading;
```

Comment:
Kan vi inline dette?

Reply:
Kan det, liker å ha det som en egen variabel, men ikke spesielt viktig for meg!